openapi/views/openapi_view.xml

```diff
@@ -0,0 +1,255 @@
+                            <field name="create_uid" readonly="1" />
+                            <field name="log_request" />
+                            <field name="log_response" />
+                            <field name="last_log_date" readonly="1" />
+                            <field name="token" />
+                            <field name="spec_url" widget="url" readonly="1" />
+                        </group>
+                    </div>
+                    <notebook>
+                        <page string="Accessable Models">
+                            <field
+                                name="access_ids"
+                                mode="tree"
+                                context="{'parent_name': name, 'active_test': False}"
+                            >
+                                <tree>
+                                    <field name="active" />
+                                    <field name="model_id" />
+                                </tree>
+                            </field>
+                        </page>
+                        <page string="Allowed Users">
+                            <field name="user_ids">
+                                <tree>
+                                    <field name="name" />
+                                </tree>
+                            </field>
+                        </page>
+                    </notebook>
+                </sheet>
+            </form>
+        </field>
+    </record>
+    <record id='openapi_namespace_model_view_tree' model='ir.ui.view'>
+        <field name="name">openapi.namespace.tree</field>
+        <field name="model">openapi.namespace</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+    <record id='openapi_access_model_view_tree' model='ir.ui.view'>
+        <field name="name">openapi.access.tree</field>
+        <field name="model">openapi.access</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="namespace_id" />
+                <field name="model" />
+            </tree>
+        </field>
+    </record>
+    <record model="ir.actions.act_window" id="namespace_list_action">
+        <field name="name">Intergrations</field>
+        <field name="res_model">openapi.namespace</field>
+        <field name="view_mode">tree,form</field>
+        <field name="help">Create and manage the namespaces.</field>
+    </record>
+    <record model="ir.actions.act_window" id="ir_model_accesses_list_action">
+        <field name="name">Accesses</field>
+        <field name="res_model">openapi.access</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[('model_id', '=', active_id)]</field>
+        <field name="context">{'default_model_id': active_id}</field>
+    </record>
+    <menuitem id="main_openapi_menu" name="OpenAPI" />
```

Comment:
Might be nice to add an icon here?
web_icon="openapi,static/description/icon.png"